modules/ROOT/pages/_partials/app-security/setting-up-the-app-security-sdk.adoc

```diff
@@ -0,0 +1,43 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+include::configuring-dev-env-app-security.adoc[]
+
+== Setting up {app-security-service} service SDK
+
+=== Importing the libraries
+
+[tabs]
+====
+
+// end::excludeDownstream[]
+Cordova::
++
+--
+
+. Install the link:https://www.npmjs.com/package/@aerogear/security[AeroGear Security] package from link:https://www.npmjs.com/[NPM^]
++
+[source,bash]
+----
+$ npm install @aerogear/security
```

Comment:
Should this include the `--save` argument?

```sh
$ npm install @aerogear/security --save
```

Reply:
:+1: